404.html

```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+permalink: /404.html
+---
+
+<div class="container">
+  <h1>Page not found.</h1>
```

Comment:
We should create an issue to have proper 404s on the main Dockstore site too actually